code-challenges.test.js

```diff
@@ -48,17 +90,21 @@ var randomNouns2 = ["temperature", "database", "chopsticks", "mango", "deduction
 
 
 
-
-
 // b) Create the function that makes the test pass.
 
 
-
-
+console.log(toCapitol(randomNoun1));
+console.log(toCapitol(randomNouns2));
+//yarn jest
 
 // --------------------3) Create a function that takes in an array of mixed data types and returns an array with ONLY NUMBERS sorted from least to greatest.
 
 // a) Create a test with expect statements for each of the variables provided.
+const sorting = str => {
```

Comment:
awesome start with defining the function. I really encourage pseudocoding your thoughts to organize a strategy for solving the problem